content/learn/06.hardware/nano-pcb-guide/nano-pcb-guide.md

```diff
@@ -0,0 +1,158 @@
+
+### VIN Min/Max
+
+The min/max voltage supply varies between boards. This is important to consider when choosing the battery source, that you do not exceed the limits to damage the board. 
+
+| Nano  | Nano Every | Nano 33 BLE | Nano 33 BLE Sense | Nano 33 IoT | Nano RP2040 Connect |
+| ----- | ---------- | ----------- | ----------------- | ----------- | ------------------- |
+| 7-12V | 7-18V      | 5-18V       | 5-18V             | 5-18V       | 5-18V               |
+
+### Battery Connection
+
+The Nano family was not designed to be battery powered, so it has no battery charging circuit, and does not have battery under voltage protections.
+
+If you want to connect a battery to a Nano, do so by using the VIN pin but you have to keep in mind VIN minimum requirements (which means a single cell battery won't be enough). Since there's no under voltage protection, a battery without a protection circuit will be damaged if it is kept connected to the Nano board even after it is completely discharged. It is thus recommended not to connect a battery directly but rather to use circuits like those in USB pattery packs that provide a stabilized 5V power supply and protect the battery from undervoltage.
+
+To connect a battery to a Nano board, you will need to use the VIN pin (refer to the VIN min-max table in the section above).
+
+![Nano battery connection.](assets/nano-external-power.png)
+
+## Pinout
+
+Nano boards largely share the placement of many pins, to make it easy for accessories to be designed for different Nano boards.
+
+### Serial Buses
+
+The Nano Family boards have serial buses attached to the following pins:
+
+| Protocol | Pins                        |
+| -------- | --------------------------- |
+| UART     | RX,TX                       |
+| SPI      | COPI(11), CIPO(12), SCK(13) |
+| I²C      | SDA(A4), SCL(A5)            |
+
+The location of these pins are located in the pinout for each board. These are found in the **Resources Section** product page of each board.
+
+## Nano Form Factor Design Files
+
+### Carrier Template
+
+![Nano carrier template file.](assets/carrier-templates.png)
+
+Below are template files for creating your custom **Printable Circuit Board (PCB)**.
+
+- [Nano Carrier PCB template (solder pads)](/resources/cad/NanoCarrier_SolderPads.zip) - an empty carrier template with solder pads. This is useful if you want to create a design where you solder the Nano directly on top of the PCB.
+- [Nano Carrier PCB template (connectors)](/resources/cad/NanoCarrier_Socket.zip) - an empty carrier template with connectors. This is useful if you want to create a design where you can attach and remove a Nano board easily.
+
+The files can be used together with various PCB design programs, such as [Altium](https://www.altium.com/) and [Eagle](https://www.autodesk.com/products/eagle/free-download) to create your own Nano accessories.
```

Comment:
Is the claim about EAGLE correct? My research indicates that these files can only be opened by Altium software.

Reply:
Uncertain at this point. We are planning to test this with altium, kicad and eagle. If it is not compatible, that statement will be removed, or alternatively, files compatible with kicad and eagle can be produced.